Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -79,6 +79,7 @@ class Warning(widget.OWWidget.Warning):
     graph_name = "scene"
     atr_types = ['attributes', 'metas', 'class_vars']
     atr_vals = {'metas': 'metas', 'attributes': 'X', 'class_vars': 'Y'}
+    row_vals = {'attributes': 'x', 'class_vars': 'y', 'metas': 'metas'}
```

Comment:
I know it's used only once (unlike `atr_vals`), but let's have it here because of symmetry.